zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/SnapshotChecksumUtil.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.snapshots.impl;
+
+import io.camunda.zeebe.snapshots.ImmutableChecksumsSFV;
+import java.util.SortedMap;
+
+final class SnapshotChecksumUtil {
+  static boolean checksumsEqual(
```

Comment:
I don't have the full context for this PR. But why not override `equals` in `SfvChecksumImpl`?